.env.template

```diff
@@ -70,16 +123,31 @@ BLACKHOLE_RD_MOUNT_REFRESH_SECONDS=200
 BLACKHOLE_WAIT_FOR_TORRENT_TIMEOUT=60
 BLACKHOLE_HISTORY_PAGE_SIZE=500
 
+#-----------------------------------------------------------------------------------------------#
+# DISCORD - BLACKHOLE, WATCHLIST, PLEX AUTHENTICATION, PLEX REQUEST, MONITOR RAM, RECLAIM SPACE #
+#-----------------------------------------------------------------------------------------------#
+
 DISCORD_ENABLED=false
 DISCORD_UPDATE_ENABLED=false
 DISCORD_WEBHOOK_URL=<discord_webhook_url>
+DISCORD_USER_TAG=<your_discord_user_id>
+TAG_ON_SUCCESS=true
+TAG_ON_FAILURE=false
```

Comment:
Consider making these a blackhole property instead of a discord property. Though not really sure where they belong because they currently only apply to blackhole, but should it remain that way?

Reply:
Hmm it would be cool to have repair added to the discord notifications if we are wanting to do the one big embed?